docs/book/component-guide/orchestrators/azureml.md

```diff
@@ -0,0 +1,143 @@
+---
+description: Orchestrating your pipelines to run on AzureML.
+---
+
+# AzureML Orchestrator
+
+[AzureML](https://azure.microsoft.com/en-us/products/machine-learning) is a
+cloud-based orchestration service provided by Microsoft, that enables 
+data scientists, machine learning engineers, and developers to build, train, 
+deploy, and manage machine learning models. It offers a comprehensive and 
+integrated environment that supports the entire machine learning lifecycle, 
+from data preparation and model development to deployment and monitoring.
+
+## When to use it
+
+You should use the AzureML orchestrator if:
+
+* you're already using Azure.
+* you're looking for a proven production-grade orchestrator.
+* you're looking for a UI in which you can track your pipeline runs.
+* you're looking for a managed solution for running your pipelines.
+
+## How it works
+
+The ZenML AzureML orchestrator implementation uses [the Python SDK v2 of 
+AzureML](https://learn.microsoft.com/en-gb/python/api/overview/azure/ai-ml-readme?view=azure-python) 
+to allow our users to build their Machine Learning pipelines. For each ZenML step,
+it creates an AzureML `CommandComponent` and brings them together in a pipeline.
+
+## How to deploy it
+
+{% hint style="info" %}
+Would you like to skip ahead and deploy a full ZenML cloud stack already,
+including an AzureML orchestrator? Check out the [in-browser stack deployment wizard](../../how-to/stack-deployment/deploy-a-cloud-stack.md),
+the [stack registration wizard](../../how-to/stack-deployment/register-a-cloud-stack.md),
+or [the ZenML Azure Terraform module](../../how-to/stack-deployment/deploy-a-cloud-stack-with-terraform.md)
+for a shortcut on how to deploy & register this stack component.
+{% endhint %}
+
+In order to use an AzureML orchestrator, you need to first 
+deploy [ZenML to the cloud](../../getting-started/deploying-zenml/README.md). 
+It would be recommended to deploy ZenML in the same region as you plan on 
+using for AzureML, but it is not necessary to do so. You must ensure that 
+you are connected to the remote ZenML server before using this stack component.
```

Comment:
Maybe add a link to "How to connect to ZenML server" ?

Reply:
Done.